### ISSUE TITLE
IOS and Voiceover usage problem - misterious buttons

### ISSUE THREAD

**Issue by tyrylu** [piwik/piwik-mobile-2#5339]
Hello.
While testing Piwik mobile 2 on an Iphone 6 with IOS 8.2 with Voiceover (the build-in screen reader), i have discovered some issues.
When going through the controls using VO navigation issues, two buttons without any label are reported. First os the very first UI element which Voiceover sees, the next is after the dashboard title. When activating one of those buttons, voiceover no longer sees any app UI and the back gesture does not work.

**Comment by tsteur** [comment]
Thx for the report!
